libnopegl/src/drawutils.c

```diff
@@ -132,11 +132,8 @@ void ngli_drawutils_print(struct canvas *canvas, int x, int y, const char *str,
 
 #define ATLAS_COLS 16
 #define ATLAS_ROWS  6
-#define ATLAS_CHAR_PAD 1 // prevent interpolation overlap issues with texture picking
-#define ATLAS_CHR_W (NGLI_FONT_W + 2 * ATLAS_CHAR_PAD)
-#define ATLAS_CHR_H (NGLI_FONT_H + 2 * ATLAS_CHAR_PAD)
-#define ATLAS_W (ATLAS_COLS * ATLAS_CHR_W)
-#define ATLAS_H (ATLAS_ROWS * ATLAS_CHR_H)
+#define ATLAS_W (ATLAS_COLS * NGLI_FONT_W)
+#define ATLAS_H (ATLAS_ROWS * NGLI_FONT_H)
```

Comment:
In the commit description, involved -> convoluted ?

Reply:
I wanted to transmit the meaning of "implicated", in the sense that it requires quite come commitment to build each intermediate state. But that's probably not correct English, so yeah I guess convoluted is fine. Changed.